### ISSUE TITLE
[Bug] "Not Interested" tag is not working again

### ISSUE THREAD

**Issue by alpasy** [jshackles/Enhanced_Steam#1074]
ES v8.6.
A bunch of [these games](http://i.imgur.com/Oh074uK.jpg) on the main steam page should be marked with "Not Interested" tag, according to [settings](http://i.imgur.com/M0O2U3h.jpg). 

Wishlist tag is [working fine](http://i.imgur.com/IITaSXn.jpg).

upd: i see now that it's caching problem: when you mark the game with "not interested" on one PC, it does not show up on another PC because Enhanced Steam stores and reads those tags from the local cache. but if you "Clear cached data" from the ES menu, then it reads the tags from Steam/user profile, and they appear all right (after page refresh).

**Comment by VladimirMangos** [comment]
It's work sometime and sometime not work for me. In random way.

**Comment by alpasy** [comment]
looks like it's [fixed](http://i.imgur.com/R4UdMlb.jpg) now.

**Comment by alpasy** [comment]
nope, still doesn't work consistently enough. 

but now i see that it's caching problem: when you mark the game with "not interested" on one PC, it does not show up on another PC because Enhanced Steam stores and reads those tags from the local cache. but if you "Clear cached data" from the ES menu, then it reads the tags from Steam/user profile, and they appear all right.

**Comment by CaptainFlint** [comment]
On my computer highlighting of "Not interested" games does not work too. I only recently installed the extension, and did not mark anything after that, but just in case I tried clearing the cache while being on the page with list of games containing a "not interested" one - no luck, it still did not highlight. BTW, the Tag option does not do anything for "Not interested" as well, I have it turned on too.

P.S. Wishlisted and owned games are both highlighted and tagged correctly.

Browser: Vivaldi 1.5.609.8 32-bit.
